### PR TITLE
designed the UI form

### LINKED ISSUE #233: Implement a react form. openapi mock server available
Implement a responsive 1 page react form with the existing site layout.

The website frontend repo can be run using its docker images on the local.
There is a working openapi doc to refer to which should make it easy to implement & test the endpoints out and implement the form. 
Documentation for development and testing with the mock server: https://mycovidconnect.stoplight.io/docs/registrations

**Additional context**
UI mock available. To be added here. remind me here if it's not.

### PR DESCRIPTION
**Pull request by GreatHayat**
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/244)

<a href="https://gitpod.io/#https://github.com/Xcov19/mycovidconnect/pull/244"></a>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,16 +14,19 @@
...
         "fibers": ">=3.1.0",
         "node-gyp": "^7.1.0",
         "node-sass": "^5.0.0",
         "react": "^16.13.1",
         "react-dom": "^16.13.1",
+        "react-feather": "^2.0.9",
         "react-google-maps": "^9.4.5",
         "react-meta-tags": "^0.7.4",
         "react-router-dom": "^5.2.0",
         "react-scripts": "^4.0.0",
```
[the remark this answers is not in the document]
Comment (codecakes) [reply package.json:28]: @GreatHayat can you take a merge from develop, test it once & push updates

File: src/components/Login.jsx [hunk 2]
```diff
@@ -1,74 +1,81 @@
...
-     * Dropdown click handler
-     * @param {*} e
-     */
-    function handleClick(e) {
-        if (drop && drop.current && drop.current.className) {
-            if (!e.target.closest(`.${drop.current.className}`) && open) {
-                setOpen(false);
-            }
-        }
+  const { logout, user, isAuthenticated, isLoading } = useAuth0();
```
Comment (codecakes) [inline src/components/Login.jsx:7]: @GreatHayat run fmt once https://github.com/Xcov19/mycovidconnect/blob/7d9fe18e6e926015f49c8ba679eb7bffe42046e7/package.json#L46

**Review by codecakes — CHANGES_REQUESTED** [review]
left some comments

**Comment by tim08094495757** [comment]
use MainHOC?

**Comment by codecakes** [comment]
issue #233 is closed. no longer the solution desired.

**Pull request closed without merging**